users/relaying.rst

```diff
@@ -38,6 +61,10 @@ smooth operation::
     Usage of relaysrv:
       -debug
             Enable debug output
+      -ext-address string
+            An optional address to advertising as being available on.
+            Allows listening on an unprivileged port with port forwarding from e.g.                                             
+            443, and be connected to on port 443.
```

Comment:
That sounds good. And if you like, you could add/keep something like the example that is there already:
e.g. :443 and then forward external port 443 to the local port that relaysrv is using.